infra/aws/terraform/iam/k8s-infra-prow-canary/terraform.tfvars

```diff
@@ -18,6 +18,7 @@ region = "us-east-2"
 
 eks_infra_admins = [
   "pkprzekwas",
-  "xmudrii"
+  "xmudrii",
+  "wozniakjan"
```

Comment:
Just a nit, can we sort this list alphabetically?

Reply:
Sure :)